### PR TITLE
drawStackedBarChart(): PHP 7.1 error

### PR DESCRIPTION
**Pull request by dehrk**
initialise LastX & LastY as null to fix array to string conversion error

### CODE REVIEW

File: src/Chart/Draw.php [hunk 1]
```diff
@@ -8051,8 +8051,8 @@ public function drawStackedBarChart(array $Format = [])
 
         $RestoreShadow = $this->Shadow;
 
-        $LastX = "";
-        $LastY = "";
+        $LastX = null;
```
Comment (szymach) [inline src/Chart/Draw.php]: Hey, at a first glance I think it would make more sense to initialize these variables as arrays, don't you think?

**Comment by dehrk** [comment]
Makes sense to me. I just looked at what other functions were doing, and they were either using null or a variable, so I went with null. But if you'd like I can update them to an empty array.

**Comment by szymach** [comment]
Yeah this library has a lot of weird typecasting and was written with PHP4 in mind, anyway. I corrected a lot of these, but it is easy to omit them in the jungle of legacy code. I will check again if returning an empty array instead of a null will not break anything and let you know this evening or tomorrow.

**Comment by szymach** [comment]
Alright, this function uses these variables only as array, so please change the initial values for them to an empty array.

**Pull request merged**